src/ReviewService/ReviewService.cs

```diff
@@ -20,6 +20,13 @@ public sealed class ReviewService<TReviewSettings> : IReviewService<TReviewSetti
 	private readonly IReviewSettingsSource<TReviewSettings> _reviewSettingsSource;
 	private readonly IList<IReviewCondition<TReviewSettings>> _reviewConditions;
 
+	/// <summary>
+	/// Initializes a new instance of the <see cref="ReviewService{TReviewSettings}"/> class.
+	/// </summary>
+	/// <param name="logger"><see cref="ILogger"/>.</param>
```

Comment:
These are probably wrong according the .Net guidelines. The text is supposed to be human readable sentences. This a good occasion to remind which parameter do what.

Reply:
Hmm were did found that? By doing some research I found that this is good. Also I feel like it's actually readable by an human when Visual Studio or any IDE formats it.

Reply:
I referring to the parameters that just contain a class reference.

Reply:
Ok I can change this. I agree that it should be more than that.

Reply:
I updated them.